Documentation/compilation.rst

```diff
@@ -16,34 +16,63 @@ In order to run SeisSol, you need to first install:
 Inital Adjustments to .bashrc
 -----------------------------
 
-Add the following lines to your .bashrc (vi ~/.bashrc).
+Create a new text file in your $HOME directory
 
 .. code-block:: bash
 
-  source /opt/intel/compiler/VERSION/bin/compilervars.sh intel64
+  cd $HOME
+  touch seissol_env.sh
+
+Open the file with your favorite text editor (vi ./seissol_env.sh) and add following lines:
+
+.. code-block:: bash
 
   # User specific aliases and functions
-  export PATH=$HOME/bin:$PATH
-  export LD_LIBRARY_PATH=$HOME/lib:$HOME/lib64:$LD_LIBRARY_PATH
+  export S3_HOME=$PWD/s3_software
```

Comment:
I think that the suggestion of the folder structure ~/{bin,lib,include} is sane, as it can be reused for multiple coding projects.

Nevertheless, this could (should?) be made configurable.

S3 is hard to understand, SEISSOL_HOME would be better imo